package.json

```diff
@@ -72,6 +72,8 @@
     "mocha": "^3.0.0",
     "null-loader": "^0.1.1",
     "phantomjs-prebuilt": "^2.1.7",
+    "postcss": "^5.2.6",
+    "postcss-loader": "^1.1.1",
```

Comment:
@stylesuxx just checked that these two deps are added automatically on the generator-side if the user answers the PostCSS prompt with yes. Could you remove them from here?